README.md

```diff
@@ -1,5 +1,5 @@
 # hebi_gazebo
-
+### NOTE: Intended for use with Gazebo versions 7 (Ubuntu 16.04) and 9 (Ubuntu 18.04) only, other configurations untested
```

Comment:
I think the NOTE should not need to say "other configurations untested". That should be clarified later (such as in line 7). The NOTE itself should be short and to the point.